lib/wrapperService.ts

```diff
@@ -227,6 +238,57 @@ export class WrapperService {
       return Promise.resolve(WrapperService.wrappers[contractName]);
     }
   }
+
+  private static defaultWrapperFilter: WrapperFilter = {
+    AbsoluteVote: true,
+    ContributionReward: true,
+    DaoCreator: true,
+    GenesisProtocol: true,
+    GlobalConstraintRegistrar: true,
+    SchemeRegistrar: true,
+    TokenCapGC: true,
+    UpgradeScheme: true,
+    VestingScheme: true,
+    VoteInOrganizationScheme: true,
+  };
+
+  private static completeWrapperFilter: WrapperFilter = {
```

Comment:
complete is a slightly confusing word here. something that indicates  no wrappers, hmm...

Reply:
I renamed them.